### ISSUE TITLE
yandex.ru: privacy

### ISSUE THREAD

**Issue by vehlwn** [uBlockOrigin/uAssets#13958]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://yandex.ru/pogoda/`

### Category

privacy

### Describe the issue

Open `https://yandex.ru/pogoda/`. Scroll down to the bottom where the section 'Мобильное приложение' is. There are buttons to download an app from Google Play and App Store which leads to tracking url of the form `https://redirect.appmetrica.yandex.com/serve/673628255453989055?c=weather_footer_desk`

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.43.0
Chromium: 103
filterset (summary): 
  network: 188916
  cosmetic: 231858
  scriptlet: 38275
  html: 0
listset (total-discarded, last updated): 
  added: 
    https://raw.githubusercontent.com/Spam404/lists/master/adblock-list.txt: 7252-0, 10m
    https://winhelp2002.mvps.org/hosts.txt: 8730-2, 10m
    RUS-0: 41885-11, 11m
    adguard-annoyance: 54964-367, 11m
    adguard-generic: 65560-899, 11m
    adguard-mobile: 7922-112, 10m
    adguard-social: 17827-875, 10m
    adguard-spyware: 26314-1127, 10m
    adguard-spyware-url: 665-13, 10m
    block-lan: 43-0, 10m
    curben-phishing: 21800-0, 10m
    curben-pup: 200-1, 10m
    dpollock-0: 10407-1482, 10m
    fanboy-annoyance: 75038-4227, 10m
    fanboy-cookiemonster: 35987-35987, 10m
    fanboy-social: 23236-23236, 10m
    fanboy-thirdparty_social: 68-21, 10m
    ublock-annoyances: 4479-232, 10m
  default: 
    user-filters: 9-0, never
    easylist: 68483-1368, 10m
    easyprivacy: 27361-10508, 10m
    plowe-0: 3647-1466, 10m
    ublock-abuse: 75-1, 10m
    ublock-badware: 4212-320, 10m
    ublock-filters: 32088-3800, 10m
    ublock-privacy: 223-49, 10m
    ublock-quick-fixes: 214-17, 10m
    ublock-unbreak: 1829-106, 10m
    urlhaus-1: 5011-1, 10m
filterset (user): [array of 10 redacted]
modifiedUserSettings: 
  contextMenuEnabled: false
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 1455 ms
  maxAssetCacheWait: 981 ms
popupPanel: 
  blocked: 66
  blockedDetails: 
    yandex.ru: 66
```
</details>

**Comment by mapx-** [comment]
it should work for all apps

```
||play.google.com/store/apps/*referrer$removeparam=referrer
||apps.apple.com/ru/app/*referrer$removeparam=referrer
```

**Comment by mapx-** [comment]
@vehlwn for russian pages add this filter list too (removing for example right side ads on the page above)

subscribe RU counters list
https://subscribe.adblockplus.org/?location=https://easylist-downloads.adblockplus.org/cntblock.txt&title=RU%20AdList:%20Counters

**Comment by krystian3w** [comment]
Dimisa use removeparam daily?

---

AdGuard Team simplfied to:

`||play.google.$removeparam=refferer`

@alex-302 @zyloden maybe similar can by added for app store.

---


@iam-py-test try test remove `c=` on `appmetrica.yandex.com`

**Comment by vehlwn** [comment]
> @vehlwn for russian pages add this filter list too (removing for example right side ads on the page above)
> 
> subscribe RU counters list https://subscribe.adblockplus.org/?location=https://easylist-downloads.adblockplus.org/cntblock.txt&title=RU%20AdList:%20Counters

Thanks. Ads on `yandex.ru` pages got blocked, but domain `redirect.appmetrica.yandex.com` is still available. Shouldn't it be considered harmful? It steals my clicks so Yandex knows who and from which page clicked on which button.

**Comment by krystian3w** [comment]
I agree we should make it harder for yandex detection form user click on link (so webmaster must add event listener on button instead simplify to url param).

**Comment by mapx-** [comment]
blocking `redirect.appmetrica.yandex.com` will break the store access too
